test/e2e/scripts/wait-for-running-branch.sh

```diff
@@ -32,6 +34,14 @@ until $(echo $STATUS | grep -wqe "Ready\|NeedsPriming" ); do
   elif [ $COUNT == $RESETCOUNT ]; then
     echo "Reached reset timeout, attempting to reset the branch"
     curl https://hash-$sha.calypso.live/?reset=true >/dev/null 2>&1
+  elif [[ "$STATUS" == "FAIL" ]] && [[ $RESETATTEMPTS < $MAXRESETATTEMPTS ]]; then
+    echo "Build Failed, attempting to reset the branch"
+    curl https://hash-$sha.calypso.live/?reset=true >/dev/null 2>&1
+    ((RESETATTEMPTS++))
+    sleep 20
```

Comment:
why the sleep 20? we don't do it for the other reset.

Reply:
Just to give the build time to restart and give a new status before we fail the build. In testing it was coming back as failed again a second time before the reset even showed up in the browser.

Reply:
ah ok. wfm.